graph_weather/models/forecast.py

```diff
@@ -29,6 +31,7 @@ def __init__(
         hidden_layers_decoder: int = 2,
         norm_type: str = "LayerNorm",
         use_checkpointing: bool = False,
+        constraint_type: str = "additive",  # "additive", "multiplicative", "softmax" or "none"
```

Comment:
I would default to "none"
```suggestion
        constraint_type: str = "none",  # "additive", "multiplicative", "softmax" or "none"
```

Reply:
Ack, forgot to update this.